### PR TITLE
Burner assault rifle buff

### PR DESCRIPTION
**Pull request by a-person5660**
## About the PR
burner rifle gets AP for shits and giggles, ammo is more available and mags are slightly bigger.

## Why / Balance
it was suggested on the discord, you kinda plow through bullets.
1.6-1.1 TC per bullet :trollface: 

image

https://discord.com/channels/1202734573247795300/1314468353783631892

**Changelog**
:cl:
- tweak: The Burner assault rifle now has AP as god intended.
- tweak: The Burner assualt rifle now has 15 round mags and you can buy more in the uplink.

### CODE REVIEW

**Comment by Piras314** [comment]
Just decrease the stupid mag cost

**Comment by JoeHammad1844** [comment]
what about 20 :troll:

**Comment by a-person5660** [comment]
actually, im gonna make it deal AP instead of 15 rounders :trollface:

**Review by Piras314 — COMMENTED** [review]
you increased the fire rate, made it AP, and decreased its cost? 😭

**Comment by a-person5660** [comment]
> you increased the fire rate, made it AP, and decreased its cost? 😭

Forgor to remove AP from the PR :trollface: 
Instead I'm leaving the price at 150 TC lol

File: Resources/Prototypes/_Goobstation/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml [hunk 1]
```diff
@@ -92,6 +92,7 @@
       types:
         Piercing: 35
         Structural: 130 # if you were a wall would you like having a .50 cal anti materiel round go through you? (this breaks a normal wall in 3 shots, one mag goes through four normal walls, 10 rounds to take down a reinforced wall)
```
Comment (TurboTrackerss14) [inline Resources/Prototypes/_Goobstation/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml:94]: Comment should likely be updated due to the "one mag goes through x" calculation. 
Could just be removed tbh for "10 rounds to take down a reinf wall"

**Review by Piras314 — CHANGES_REQUESTED** [review]
Remove AP alltogether.

**Pull request merged**